### PR TITLE
Madiha-week2 exercises 

### PR DESCRIPTION
**Pull request by diya2214**

### CODE REVIEW

File: week-2/F-logical-operators/exercise2.js [hunk 1]
```diff
@@ -5,17 +5,29 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(number) {
+  return number <0 ? true : false;
```
Comment (laperla11) [inline week-2/F-logical-operators/exercise2.js]: Good use of ternary operators. well done.

File: week-2-practise/level-3/6-map.js [hunk 2]
```diff
@@ -11,11 +11,14 @@ function tidyUpString(str) {
...
 function captialise(str) {
+  str = str.charAt(0).toUpperCase()+str.substr(1);
+  return str;
   // complete this function
 }
 
 var mentors = ["/Daniel ", "irina ", " Gordon", "ashleigh "];
-var mentorsTidyAndCapitalised;
+var mentorsTidy = mentors.map(tidyUpString);
+var mentorsTidyAndCapitalised = mentorsTidy.map(captialise);
```
Comment (LucyMac) [inline week-2-practise/level-3/6-map.js]: Same as previous comment about .map() format

File: week-2/B-boolean-literals/exercise.js [hunk 3]
```diff
@@ -6,7 +6,9 @@
 */
 
 var codeYourFutureIsGreat = true;
-
+mozafarIsCool = false;
+caluclationCorrect = true;
+moreThan10Students = false;
```
Comment (LucyMac) [inline week-2/B-boolean-literals/exercise.js]: Don't forget the `var` keyword when defining a variable for the first time!

File: week-2/F-logical-operators/exercise.js [hunk 4]
```diff
@@ -12,25 +12,41 @@ var cssLevel = 4;
...
-// Finish the next two statement
+// Finish the next  two statement
 // Use the previous variables and logical operators
 // Do not "hardcode" the answers
 var cssAndHtmlAbove5;
+if (cssLevelAbove5 === true && htmlLevelAbove5 === true) {
+  cssAndHtmlAbove5 = true;
+} else {
+  cssAndHtmlAbove5 = false;
+}
```
Comment (LucyMac) [inline week-2/F-logical-operators/exercise.js]: Can you do the same thing without using an `if/else` statement? 😛

File: week-2/J-array-get-set/exercise.js [hunk 5]
```diff
@@ -5,11 +5,13 @@
...
 function first(arr) {
-  return; // complete this statement
+  return arr[0]; // complete this statement
 }
 
 function last(arr) {
-  return; // complete this statement
+  lengthArr = arr.length;
+
+  return arr[lengthArr - 1]; // complete this statement
```
Comment (LucyMac) [inline week-2/J-array-get-set/exercise.js]: Good

File: week-2/M-array-map/exercise.js [hunk 6]
```diff
@@ -10,7 +10,7 @@ function double(num) {
 }
 
 var numbers = [1, 2, 3, 4];
-var numbersDoubled; // complete this statement (use map and the double function)
+var numbersDoubled = numbers.map(double);
```
Comment (LucyMac) [inline week-2/M-array-map/exercise.js]: Same issue with map()...

File: week-2-practise/level-1/2-validation.js [hunk 7]
```diff
@@ -2,7 +2,7 @@
 // TIP: You should write an expression that returns a boolean value
 
 var num = 10;
-var isBigEnough; // ONLY EDIT THIS LINE
+var isBigEnough = true; // ONLY EDIT THIS LINE
```
Comment (40thieves) [inline week-2-practise/level-1/2-validation.js]: I don't think this is quite the correct answer. The exercise is asking you to calculate if `num` is bigger or equal to 10. What happens if we change the `num` variable to 12?

File: week-2/F-logical-operators/exercise2.js (hunk 1 above)
Comment (mtbdata711) [reply week-2/F-logical-operators/exercise2.js]: Seconding Kadir's point here. I know one day we will all have to learn how to do these in our everyday code but it still makes no sense to me! Very impressed at how clean it makes everything look.

File: week-2/F-logical-operators/exercise.js (hunk 4 above)
Comment (khan-zareen) [reply week-2/F-logical-operators/exercise.js]: Good use of logical operator

**Pull request closed without merging**